rules/integrations/o365/initial_access_microsoft_365_impossible_travel_portal_logins.toml

```
@@ -22,58 +22,75 @@ from = "now-15m"
-- Consider time zone differences that might affect the perceived timing of logins. Adjust the rule's sensitivity to account for legitimate time zone shifts that could appear as impossible travel.
+- Users traveling or using VPNs may trigger this alert. Verify with the user if they were traveling or using a VPN at the time of the login attempt.
+- Mobile access may also result in false positives, as users may log in from various locations while on the go.
 
 ### Response and remediation
 
-- Immediately isolate the affected user account by disabling it to prevent further unauthorized access.
-- Initiate a password reset for the compromised account and enforce multi-factor authentication (MFA) to enhance security.
-- Review recent login activity and audit logs for the affected account to identify any unauthorized access or data exfiltration attempts.
-- Notify the user of the suspicious activity and advise them to verify any recent changes or actions taken on their account.
-- Escalate the incident to the security operations team for further investigation and to determine if other accounts or systems have been compromised.
-- Implement geo-blocking for high-risk countries or regions where the organization does not typically conduct business to prevent similar unauthorized access attempts.
-- Update and refine security monitoring rules to enhance detection of similar anomalous login patterns in the future."""
+- Investigate the login attempt further by checking for any additional context or related events that may provide insight into the user's behavior.
+- If the login attempt is deemed suspicious, consider implementing additional security measures, such as requiring multi-factor authentication (MFA) for logins from unusual locations.
+- Educate users about the risks of accessing corporate resources from unfamiliar locations and the importance of using secure connections (e.g., VPNs) when doing so.
+- Monitor for any subsequent login attempts from the same location or IP address to identify potential patterns of malicious activity.
+"""
+references = ["https://www.huntress.com/blog/time-travelers-busted-how-to-detect-impossible-travel-"]
+risk_score = 47
+rule_id = "3896d4c0-6ad1-11ef-8c7b-f661ea17fbcc"
+severity = "medium"
+tags = [
+    "Domain: Cloud",
+    "Domain: SaaS",
+    "Data Source: Microsoft 365",
+    "Data Source: Microsoft 365 Audit Logs",
+    "Use Case: Threat Detection",
+    "Use Case: Identity and Access Audit",
+    "Tactic: Initial Access",
+    "Resources: Investigation Guide",
+]
+timestamp_override = "event.ingested"
+type = "threshold"
+
+query = '''
+event.dataset:o365.audit and
+    event.provider:AzureActiveDirectory and
+    event.action:UserLoggedIn and
+    event.outcome:success and
+    o365.audit.Target.Type:(0 or 10 or 2 or 3 or 5 or 6) and
+    o365.audit.UserId:(not "Not Available" and *) and
```

make it easy to read/understand
```suggestion
    o365.audit.UserId:(* and not "Not Available") and
```